src/common/resourceLoaders/apiTreeLoader.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import fs from "fs-extra";
+import path from "path";
+import { createApi } from "../structures/api";
+import { ApiMetadata } from "../structures/apiMetadata";
+import { ApiTree } from "../structures/apiTree";
+
+/**
+ * Given a directory parse all apis and their metadata in those directories
+ *
+ * @param apiPath - The path to the root of the apis.
+ */
+export async function createApiTree(apiPath: string): Promise<ApiTree> {
```

Comment:
This function is labeled as async, but it doesn't do anything async other than call itself. Either convert this to sync or use async `fs` methods.

Reply:
Create API is async.  Which is why this is.